vnext/ReactWindowsCore/IWebSocketResource.h

```diff
@@ -129,7 +136,7 @@ struct IWebSocketResource {
   /// </param>
   /// <param name="reason">
   /// </param>
-  virtual void Close(CloseCode code, const std::string &reason) = 0;
+  virtual void Close(CloseCode code = CloseCode::None, const std::string &reason = {}) = 0;
```

Comment:
>Close [](start = 15, length = 5)

Julio, for next changes here, can you start new branch or squash it? It's really difficult for code reviewers to keep track of all these iterations, currently 113. Thank you.

Reply:
Note, these are commits and not pushes or "iterations".
I'd prefer to keep the full history of changes in the personal branches (which can some times be finely granular).

We can discuss consensus on this offline.